packages/workspace/src/workspace/workspace.ts

```diff
@@ -910,6 +932,23 @@ export const loadWorkspace = async (
     ?? []
     return result.filter(values.isDefined).flat()
   }
+  const getChangedElementsBetween = async (
+    dateRange: DateRange,
+    envName?: string,
+  ): Promise<ElemID[]> => {
+    const isDateInRange = (date: string): boolean => {
+      const dateToCheck = new Date(date)
+      return dateToCheck >= dateRange.start
+      && (isDefined(dateRange.end) ? dateToCheck <= dateRange.end : true)
```

Comment:
nit: I think the if condition here is a bit confusing, maybe:
```suggestion
      && (dateRange.end === undefined || dateToCheck <= dateRange.end)
```